auth/src/main/java/com/firebase/ui/auth/ui/email/RecoveryEmailSentDialog.java

```diff
@@ -13,14 +13,16 @@
 import com.firebase.ui.auth.ResultCodes;
 import com.firebase.ui.auth.ui.DialogBase;
 import com.firebase.ui.auth.ui.ExtraConstants;
+import com.firebase.ui.auth.ui.FlowParameters;
 
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 public class RecoveryEmailSentDialog extends DialogBase {
     private static final String TAG = "RecoveryEmailSentDialog";
 
-    public static void show(String email, FragmentManager manager) {
+    public static void show(String email, FlowParameters parameters, FragmentManager manager) {
         RecoveryEmailSentDialog result = new RecoveryEmailSentDialog();
         Bundle bundle = new Bundle();
+        bundle.putParcelable(ExtraConstants.EXTRA_FLOW_PARAMS, parameters);
```

Comment:
Oh, just noticed you added the flow params. Yeah, lol. We should just kill `DialogBase` since we aren't using any of it and it's forcing us to add dead code to the `RecoveryEmailSentDialog`.